src/data/abbreviations.js

```diff
@@ -48,6 +48,7 @@ let compact = {
   ],
 
   Place: [
+    'Str',
```

Comment:
this would usually be abbreviated with `Str.` would the dot be normalized away by compromise?